src/Symfony/Component/Mime/Tests/MessageTest.php

```diff
@@ -131,7 +131,11 @@ public function testToString()
 
 EOF;
         $this->assertStringMatchesFormat($expected, str_replace("\r\n", "\n", $message->toString()));
-        $this->assertStringMatchesFormat($expected, str_replace("\r\n", "\n", implode('', iterator_to_array($message->toIterable()))));
```

Comment:
All you need to do here instead is to just set second argument as `false` in `iterator_to_array`

Reply:
indeed, thank you!